superset-frontend/src/features/alerts/AlertReportModal.test.tsx

```diff
@@ -413,6 +420,21 @@ test('renders screenshot options when dashboard is selected', async () => {
   ).toBeInTheDocument();
 });
 
+test('renders tab selection when Dashboard is selected', async () => {
+  render(<AlertReportModal {...generateMockedProps(false, true, true)} />, {
+    useRedux: true,
+  });
+  userEvent.click(screen.getByTestId('contents-panel'));
+  await screen.findByText(/test dashboard/i);
+  expect(
+    screen.getByRole('combobox', { name: /select content type/i }),
+  ).toBeInTheDocument();
+  expect(
+    screen.getByRole('combobox', { name: /dashboard/i }),
+  ).toBeInTheDocument();
+  expect(screen.getByText(/select tab/i)).toBeInTheDocument();
```

Comment:
Should we verify that the TreeSelect renders and has some options?

Reply:
None of the other select options are being rendered for owners, dashboards, charts, or tabs. I haven't been able to get options to show for antd select components within the RTL framework. Is it worth trying another testing framework for this?

Reply:
I think it's ok for now